Quick Fix

IceNWinter opened the pull request:
*changed the name of the elegant receptor to extreme receptor as they changed it global name
*fixed the effect text of the gift receptor as it was formated wrong

IceNWinter committed:
Merge pull request #2 from CorVous/master

updating files

IceNWinter committed:
Updating Translations

Changed a few global translations with Special Ability to Augment
Removed the * from Sentence Triple as that is currently a augment in Global Release

IceNWinter committed:
Translations Update

update a few translations and un-greyed them after discovering them in-game for their global translation

IceNWinter committed:
reuploading fix

reuploaded files as was using master files from an older version

IceNWinter committed:
updating master files

updated master files here to match fork master

IceNWinter committed:
S-Grade Update

Updated global translations for the currently known S-Grade augments that been added
Also added the updated updatemanager.min.js file from the original repository by malulleybovo

IceNWinter committed:
S-Grade Update 2

Went back in to fix some translations from the first S-Grade Update
This also include greying out a few of the ungreyed s-grade augments as i learn there not obtainable yet

IceNWinter committed:
S-Grade Update 3

a few more small fixes

IceNWinter committed:
Merge branch 'master' into update

IceNWinter committed:
Update, new augs

*Translated a few reverie augs and ungreyed them as they been added to the global verison
*translated the guidance trainer to the global translation of augment guidance

IceNWinter committed:
quick fix

*ungreyed apprentice reverie augment

IceNWinter committed:
another quick fix

*chanced the guidance trainer +5% to augment guidance +5%

IceNWinter committed:
PSO2 AA Update

*Updated translations of S-Grade Augments that had a renaming for global
*Updated translations of new augments added to global with their global translation

IceNWinter committed:
quick updated

*ungreyed the lesser augments

IceNWinter committed:
Merge branch 'master' into update

IceNWinter committed:
Big Augment Update

Added in the translation of many augments with their global translation

IceNWinter committed:
fixing up some formating

due to how the use of comma changes the format of the effect text and messing up the text, I went back and fixed some of the text with that issue

IceNWinter committed:
another quick format fix

IceNWinter committed:
Merge branch 'master' into update

IceNWinter committed:
ungreyed Grand augments

forgot to ungrey grand augments that are now available in game

IceNWinter committed:
Merge branch 'master' into update

IceNWinter committed:
Some final touch ups

found some mistakes made in the translations and updated some translations with their global version and ungreyed them

IceNWinter committed:
Fixed translations found by sgosiaco

Thanks to sqosiaco, found and fix the following translations:

    Changed Soverign to Sovereign
    Changed Tiro to Tyro (For Sentence Tyro)
    Changed Magia Di to Magi Di
    Changed Sentence Deftness to Sentence Arma
    Changed Greuzoras to Gryzorus
    Changed Duvals to Deubarz

IceNWinter committed:
Merge branch 'master' into update

IceNWinter committed:
Global EP6 Augment Update

Updated new available augments that made it to the global version with their global translations and or now ungreyed

IceNWinter committed:
Merge branch 'master' into update

IceNWinter committed:
quick update

updated and un-greyed Phrase Decay and Slayer: Falspawn augments

IceNWinter committed:
One to One Update

* Added in all augments missing from this version of the Affix Assissant that have been added to malulleybovo original repository
* Updated those and other recently added augments global translations available in the global version of PSO2
* Added in all bug fixes made in the original repository to the global translation version of it

IceNWinter committed:
Merge branch 'master' into update

IceNWinter committed:
Jan 1/20 Update

*change the remaining global effect text with jp translations to global translations as the mix of the two doesn't behave correctly in the sim
*updated global translations and ungreying more translations

IceNWinter committed:
Merge branch 'master' into update

IceNWinter committed:
touch up updated

*updating a few augments translations for global

IceNWinter committed:
Merge branch 'master' into update

IceNWinter committed:
More translation to the pile Update

*Updated the translations and un-greyed them as been added to global version
*all S4 augments has been translated

IceNWinter committed:
Merge branch 'master' into update

IceNWinter committed:
hotfix

*ungreyed S7: Fortified Life Vessel augment

IceNWinter committed:
Guardian Soul Update

*Updated and ungrey the rest of the glare augments and also guardian soul augment as global has all the augment needed to make it
*Updated the rest of the S5 augments
*Ungrey the rest of the augment capsulas leaving only s grade augments not in global yet

IceNWinter committed:
Merge branch 'master' into update

IceNWinter committed:
Quick Fix

*changed the name of the elegant receptor to extreme receptor as they changed it global name
*fixed the effect text of the gift receptor as it was formated wrong

IceNWinter committed:
uploading the right version of the files

IceNWinter committed:
uploading the correct verison that has been built

IceNWinter committed:
Add files via upload

IceNWinter committed:
Merge branch 'master' into update

PaJson commented:
I sent an e-mail about name_glen: Light Resist should be corrected to Radiant Ward, but seems this was missed. Maybe for next update?

IceNWinter committed:
V1.2.3 Update

*added a +20% option in the boost week option drop-down menu

IceNWinter committed:
translation fix

*Changed the global name of the augments with the name Light Resist to it correct global name of Radiant Ward, thanks to PaJson for discovering this issue
